crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -847,6 +857,67 @@ pub(crate) fn job_progress_bar(
     buffer
 }
 
+pub fn print_job_output(
+    job: JobDetail,
+    task_selector: Selector,
+    output_stream: OutputStream,
+) -> anyhow::Result<()> {
+    fn read_stream(task_state: &JobTaskState, output_stream: &OutputStream) {
+        let (_, stdout, stderr) = get_task_paths(task_state);
+        let path = match output_stream {
+            OutputStream::Stdout => stdout,
+            OutputStream::Stderr => stderr,
+        };
+
+        match File::open(path) {
+            Ok(mut file) => {
+                let copy = std::io::copy(&mut file, &mut std::io::stdout().lock());
+                if let Err(error) = copy {
+                    log::warn!("File: {} cannot be copied: {:?}", path, error);
```

Comment:
Since recently, you can use identifiers in formatting placeholders directly :)
`File: {path} cannot be copied: {error:?}`